judgments/utils/search_utils.py

```diff
@@ -0,0 +1,19 @@
+from ds_caselaw_utils import courts as all_courts
+from ds_caselaw_utils.courts import CourtNotFoundException
+
+
+def process_facets(facets: dict):
+    """
+        Util for processing court facets
+    """
+    unprocessed_facets = facets.copy()
+    court_facets = {}
+    for facet_key, facet_value in facets.items():
+        try:
+            court_code = all_courts.get_by_code(facet_key)
```

Comment:
I think this could be a lot simpler if we got all the court codes ourselves: we could maybe write something like:

court_codes = [court.get("code") for court in all_courts]
valid_facets = [facet for facet in facet if facet_key in court_codes]
invalid_facets = [facet for facet in facet if facet_key not in court_codes]
return invalid_facets, valid_facets